### PR TITLE
[plugins] Let a plugin author pointed out where the plugin hosted

### PR DESCRIPTION
**Pull request by DmitrySharabin**

### CODE REVIEW

File: src/plugins.js [hunk 1]
```diff
@@ -34,7 +34,11 @@ var _ = Mavo.Plugins = {
 				// Load plugin
 				var filename = `mavo-${plugin.id}.js`;
 
-				if (plugin.repo) {
+				if (plugin.url) {
+					// The plugin author pointed out where the plugin hosted
```
Comment (LeaVerou) [inline src/plugins.js:38]: Grammar: "where the plugin is hosted"

File: src/plugins.js [hunk 2]
```diff
@@ -34,7 +34,11 @@ var _ = Mavo.Plugins = {
...
+					var url = plugin.url;
```
Comment (LeaVerou) [inline src/plugins.js:39]: Let's try and gradually convert `var` to `let` when we touch a piece of code. Especially things like this, which are taking advantage of the fact that `var` is function scope, and will need manual conversion anyway.

**Review by LeaVerou — CHANGES_REQUESTED** [review]
Thanks Dmitry, LGTM, see my two minor comments.

File: src/plugins.js (hunk 1 above)
Comment (DmitrySharabin) [reply src/plugins.js:38]: Thank you! Fixed.

File: src/plugins.js (hunk 2 above)
Comment (DmitrySharabin) [reply src/plugins.js:39]: Agreed.

File: src/plugins.js [hunk 3]
```diff
@@ -32,15 +32,20 @@ var _ = Mavo.Plugins = {
...
 
 				// Load plugin
-				var filename = `mavo-${plugin.id}.js`;
+				let filename = `mavo-${plugin.id}.js`;
+				let url;
 
-				if (plugin.repo) {
+				if (plugin.url) {
+					// The plugin author pointed out where the plugin is hosted
+					url = plugin.url;
```
Comment (DmitrySharabin) [inline src/plugins.js:40]: If we come to the conclusion that `plugin.url` is a base URL for all the plugin files and is not include `filename`, I'll replace this line with:
```js
url = `${plugin.url}/${filename}`
```

Comment (LeaVerou) [reply src/plugins.js:40]: Was there any further discussion on that?

Comment (DmitrySharabin) [reply src/plugins.js:40]: Yep, it's here: https://github.com/mavoweb/plugins/pull/85#discussion_r569444932

**Comment by DmitrySharabin** [comment]
We'll be back to this question later. For now, I'm going to close it.

**Pull request closed without merging**